node/src/components/consensus/protocols/highway/tests.rs

```diff
@@ -151,6 +151,68 @@ fn send_a_wire_unit_with_too_small_a_round_exp() {
     }
 }
 
+#[test]
+fn send_a_valid_wire_unit() {
+    let creator: ValidatorIndex = ValidatorIndex(0);
+    let state: State<ClContext> = new_test_state(&[100.into()], 0);
+    let panorama: ValidatorMap<Observation<ClContext>> = Panorama::from(vec![N, N]);
```

Comment:
```suggestion
    let panorama: Panorama<ClContext> = Panorama::from(vec![N, N]);
```
(Except if you did that deliberately, to be more explicit?)